charts/osm/crds/multiclusterservice.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Could we add comments similar to this : https://github.com/openservicemesh/osm/blob/b223d9ed3fb50ce6fcc09842d1bba5fba6238482/charts/osm/crds/policy.yaml#L1